motion_tracker/utils/crop_vis.py

```diff
@@ -41,3 +41,18 @@ def show_stages_of_random_crop(img, box_coords, output_width=256, output_height=
                                                   output_width, output_height)
     add_box(final_img, box_coords)
     show_img("Random Crop", final_img)
+
+def show_single_stage(img, box_coords, title="Happy Dance Image"): 
```

Comment:
After the current PR is merged, I'd like to get rid of the current show_img, and replace it with something like this.  We can give box_coords a default argument of None, and only plot the box if box_coords isn't None.

That's basically this function... and having it obviates the need for add_box and show_img.

It also simplifies a bunch of places where I sequentially called add_box and show_img.

It's not like we are ever calling add_box and then not showing the image :)